HW03 Brock.Rmd

```diff
@@ -0,0 +1,26 @@
+---
+title: "COVID Mapping"
+author: "Brock Adams"
+date: "7/20/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(readr) #reading rectangular text data
+library(ggplot2) #plotting data
+library(magrittr) #piping
+```
+
+```{r COVID Data import}
+COVID_url = "https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv"
```

Comment:
Thanks for providing the source of your data; it's really helpful! If you wanted to make it a clickable link in the future, you can use the code: [COVID_url](https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv).

Reply:
Oh oops you can't see the code because it actually made it a link, but basically it is "name of link" in square brackets with the link next to it in parenthesis.